### DISCUSSION TITLE
Please remove this religious hate-speech

### DISCUSSION THREAD

**Discussion by dftf-stu** [community/community#114716] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Please remove the following hate-speech:

> _Dirty corporations and Hindu-managers are trying to deprive the any users of all the benefits that were created by technically more advanced people than them. Windows 7 and Windows XP are the most successful and best technical creations of Microsoft ever._

Which can be found here, under the "Intro" heading:

https://github.com/Blaukovitch/GOOGLE_CHROME_Windows_7_CRACK

Thank you

[Edit: having to write this here as the ticket was closed!]

Hi Lili (@queenofcorgis)

Already reported it that-way about a month ago, and also on the "Report Hate Speech Content Posted to a Microsoft Hosted Consumer Service" website, here: https://www.microsoft.com/en-us/concern/hatespeech

Any ETAs on how long it takes to be actioned?

Thank you!

**Comment by queenofcorgis** [comment, marked as answer]
Hi @dftf-stu 👋🏼  We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our abuse reporting tools. Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. I am going to close this post, but for this and any future incidents, please refer to those links.

Thank you!
